src/Component/Cart/Cart.js

```diff
@@ -0,0 +1,118 @@
+import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import "./Cart.scss";
+import EmptyTable from "./Component//EmptyTable/EmptyTable";
+import ProductTable from "./Component/ProductTable/ProductTable";
+import FillAsidebar from "./Component/Asidebar/FillAsidebar";
+import EmptyAsidebar from "./Component/Asidebar/EmptyAsidebar";
+// import Navbar from "../../Component/Navbar/NavbarStore/"
+import Footer from "./Component/Footer/Footer";
+
+// const API =
+class Cart extends Component {
+  constructor() {
+    super();
+    this.state = {
+      cartList: [],
+      isChecked: true,
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/cartItem.json", {
+      method: "GET",
+    })
+      .then((res) => res.json())
+      .then((res) => {
+        this.setState({
+          cartList: res.item,
+        });
+      });
+  }
+
+  handlePlusBtn = (item) => {
+    const cartList = [...this.state.cartList];
+    let idx = cartList.indexOf(item);
+    if (cartList[idx].quantity < 10) {
+      cartList[idx].quantity++;
+      this.setState({ cartList });
+    } else {
+      alert("최대 주문 수량은 10개입니다");
+    }
+  };
+
+  handleMinusBtn = (item) => {
+    const cartList = [...this.state.cartList];
+    let idx = cartList.indexOf(item);
+    if (cartList[idx].quantity > 1) {
+      cartList[idx].quantity--;
+      this.setState({ cartList });
+    } else {
+      return;
+    }
+  };
+
+  render() {
+    const { cartList, isChecked } = this.state;
+    const { handlePlusBtn, handleMinusBtn } = this;
+
+    return (
+      <>
+        {/* <Navbar /> */}
+        <div className="cart">
+          <div className="allContainer">
+            <div className="headContainer">
+              <div className="title">
+                <h2>장바구니</h2>
+              </div>
+              <div className="process">
+                <span>장바구니 {">"} </span>
+                <span>&nbsp;주문결제 {">"} 주문완료</span>
+              </div>
+            </div>
+            <div className="mainContainer">
+              <div className="cartContainer">
+                <div className="header">
+                  <div className="headerLeft">
+                    <input type="checkbox" onClick="CheckAll" />
+                    <span onClick="checkAll">전체선택</span>
+                  </div>
+                  <div className="headerRight">
+                    <button>삭제</button>
+                  </div>
+                </div>
+                {cartList.length < 1 ? (
```

Comment:
cartLIst의 length가 -일리는 없고, 0일 경우를 나타내는 것 같아요! 
그럼 boolean값을 이용해서 아래와 같이 작성해주실 수 있겠네요! 🤗
```suggestion
                {!cartList.length ? (
```

Reply:
오 역시.......감사합니다!